### ISSUE TITLE
Problem after upgrade

### ISSUE THREAD

**Issue by martinezpenya** [wernight/docker-mopidy#34]
I've been using yout docker-mopidy for at least 5 month without problems, from webserver, android client and home assistant.
Two days ago I've pull new image (includes iris, and is pretty), but home assistant stop working and I can only play music from browser on port 6680.
I can't see any error, but something has changed. Can you help me?

**Comment by martinezpenya** [comment]
If I run this command on the docker
`python3 -m pip install Mopidy-MPD`
And restart the mopidy it works again... I don't understand what's happening
Why is removed the mpd service from de mopidy container?

**Comment by ahmadnbl** [comment]
yep @martinezpenya  that command makes the Mopidy works with my home assistant setup.
Using default port `6600`, `Mopidy` dan `HA` on different container, and with that script it can play audio form `HA`

**Comment by sasgoose** [comment]
when I try to run this I get an error that python3 does not exist. does it have to exist within the container or on the host machine? if I just run pip install mopidy-mpd i get another error
```

DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Collecting mopidy-mpd
  Could not find a version that satisfies the requirement mopidy-mpd (from versions: )
No matching distribution found for mopidy-mpd
You are using pip version 19.0.3, however version 20.3.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```
